src/engine/sync/synccontrol.cpp

```diff
@@ -438,6 +444,18 @@ void SyncControl::setLocalBpm(double local_bpm) {
     }
 }
 
+void SyncControl::updateAudible() {
+    int channelIndex = m_pChannel->getChannelIndex();
+    if (channelIndex >= 0) {
+        CSAMPLE_GAIN gain = getEngineMaster()->getMasterGain(channelIndex);
+        bool newAudible = gain > CSAMPLE_GAIN_ZERO ? true : false;
+        if (m_audible != newAudible) {
+            m_audible = newAudible;
+            m_pEngineSync->notifyPlaying(this, m_pPlayButton->toBool() && m_audible);
```

Comment:
Same here semantic change that causes an inconsistency. Audible, Playing or a mixture of both?